src/components/game-ui/game-characteristic.vue

```
@@ -0,0 +1,349 @@
+          RadixIconsCross2
+        vs-button.accept(icon color="#2ECC71" v-if="has_pending_allocated_stats()" @click="validate_stats")
+          FluentCheckmark12Regular
+</template>
+
+<script setup>
+import { ref, inject, onMounted } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+import { get_max_health, get_total_stat } from '@aresrpg/aresrpg-sdk/stats';
+import { sui_add_stats } from '../../core/sui/client.js';
+
+import {
+  context,
+  current_locked_character,
+} from '../../core/game/game.js';
+
+import {
+  decrease_loading,
+  increase_loading,
+} from '../../core/utils/loading.js';
+
+// @ts-ignore
+import RadixIconsCross2 from '~icons/radix-icons/cross-2';
+// @ts-ignore
+import FluentCheckmark12Regular from '~icons/fluent/checkmark-12-regular';
+
+const selected_character = inject('selected_character');
+const character = current_locked_character(context.get_state());
+const supposed_max_health = character?._type ? get_max_health(character) : -1;
+
+const { t } = useI18n();
+
+const accept_loading = ref(false);
+
+const stats = ref({
+  vitality: {
+    label: t('APP_ITEM_VITALITY'),
+    imagePath: new URL('../../assets/statistics/vitality.png', import.meta.url).href
+  },
+  wisdom: {
+    label: t('APP_ITEM_WISDOM'),
+    imagePath: new URL('../../assets/statistics/wisdom.png', import.meta.url).href
+  },
+  strength: {
+    label: t('APP_ITEM_STRENGTH'),
+    imagePath: new URL('../../assets/statistics/strength.png', import.meta.url).href
+  },
+  intelligence: {
+    label: t('APP_ITEM_INTELLIGENCE'),
+    imagePath: new URL('../../assets/statistics/intelligence.png', import.meta.url).href
+  },
+  chance: {
+    label: t('APP_ITEM_CHANCE'),
+    imagePath: new URL('../../assets/statistics/chance.png', import.meta.url).href
+  },
+  agility: {
+    label: t('APP_ITEM_AGILITY'),
+    imagePath: new URL('../../assets/statistics/agility.png', import.meta.url).href
+  }
+})
+
+const pa = ref(12);
+const pm = ref(6);
```

such default could be misleading, better to show nothing, 0,  or a loading skeleton